packages/3id-did-resolver/src/index.ts

```diff
@@ -92,23 +92,21 @@ const legacyResolve = async (ceramic: Ceramic, didId: string, version?: string):
   const legacyDoc = wrapDocument(legacyPublicKeys, `did:3:${didId}`)
   if (version === '0') return legacyDoc
 
-  const managementKey = `did:key:${legacyPublicKeys.publicKeys.signing}`
-
   try {
-    const content =  { metadata: { controllers: [managementKey], family: '3id' } }
+    const content =  { deterministic: true, metadata: { controllers: [legacyPublicKeys.keyDid], family: '3id' } }
     const doc = await ceramic.createDocument('tile', content, { applyOnly: true })
-    const didDoc = await resolve(ceramic, doc.id.toString(), version)
+    const didDoc = await resolve(ceramic, doc.id.toString(), version, didId)
```

Comment:
We can inline variable here

Reply:
inline what variable?

Reply:
a minor refactor, just `return resolve(ceramic, doc.id.toString(), version, didId)`

Reply:
Aha, yep. Didn't get it because your comment only showed the line above the return statement.

Reply:
Actually we don't want to return this promise since we want to catch any error that happens while it executes.

Reply:
np! actually, it's my bad. It should be `return await resolve(ceramic, doc.id.toString(), version, didId)` 🙂